src/javascript/app/base/footer.js

```diff
@@ -1,4 +1,23 @@
+const BinarySocket = require('./socket');
+const Client       = require('../base/client');
+const State        = require('../../_common/storage').State;
+
 const Footer = (() => {
+    const onLoad = () => {
+        BinarySocket.wait('website_status', 'authorize').then(() => {
+            const $footer      = $('#footer-last');
+            const $cfd_warning = $footer.find('.eu-only');
+            const show_warning = Client.get('is_eu') || /maltainvest/.test(State.getResponse('authorize.landing_company_name'));
```

Comment:
do we want to check is_eu for logged out else check maltainvest for logged in? currently this will show for virtual and MLT clients too, not sure if we want that or not. could check with compliance.